test/viper/ok/assertions.silicon.ok

```diff
@@ -1,2 +1,4 @@
   [0] Postcondition of __init__ might not hold. Assertion $Self.u might not hold. (assertions.vpr@10.13--10.24)
+
```

Comment:
@rvanasa newlines appear with the most recent nightly, but I don't think this will influence the LSP error messages. Just a heads-up...

Reply:
They are gone again (stable `viper server.jar`).